scipy-1.0/paper.tex

```diff
@@ -193,7 +193,7 @@ \section*{Introduction}
 esri-revenue,bloom-wolfram}---was preposterous.
 
 Yet, the philosophical motivations behind a fully open toolstack, and
-an excited, friendly community with a singular focus, seemed to have
+an excited, friendly community with a singular focus, have
```

Comment:
I agree with change. But now that you highlight this sentence, the original comma usage is probably wrong, so it would be helpful if you could fix that in this PR. 
Specifically, the comma after yet seems inappropriate: https://english.stackexchange.com/questions/106901/yet-at-the-beginning-of-a-sentence
And the commas surrounding ", and an excited, friendly community with a singular focus," would only be appropriate if this is a parenthetical clause, which I don't think it is. 
The fact that:
"Yet the philosophical motivations behind a fully open toolstack and an excited, friendly community with a singular focus have proven auspicious in the long run."
looks a bit strange does not mean that the commas were used correctly in the first place. It might indicate that the sentence needs to be revised more substantially, though.

Would you mind researching this and either remove the offending commas or argue to keep them? Thank you!

Reply:
@mdhaber very good, fixed! As a minor grammar nazi, I appreciate the prod. =P